sources/bot/modules/spreadsheet/study_staff/study_staff_spreadsheet_handler.py

```diff
@@ -0,0 +1,56 @@
+from typing import List
+
+from bot.exceptions import InvalidSpreadsheetAttributeException
+from bot.modules.spreadsheet.spreadsheet_handler import SpreadsheetHandler
+
+
+class StudyStaffSpreadsheetHandler:
+    def __init__(self, spreadsheet_id: str, file_name: str):
+        _attributes = {
+            "Студенты": ["username", "ФИО", "Группа", "Подгруппа"],
+            "Преподаватели": ["username", "ФИО"],
+        }
+        self._handler = SpreadsheetHandler(spreadsheet_id, file_name, _attributes)
+        self._student_sheet_title = list(_attributes.keys())[0]
+        self._teacher_sheet_title = list(_attributes.keys())[1]
+
+    def create_spreadsheet(self, spreadsheet_title="Информация о людях", row_count=1000, column_count=10) -> None:
+        self._handler.create_spreadsheet(spreadsheet_title, row_count, column_count)
+
+    def add_student(self, username: str, *args) -> None:
+        name, group, subgroup = args
+
+        if not name:
+            raise InvalidSpreadsheetAttributeException("Invalid name value")
+        elif not group:
+            raise InvalidSpreadsheetAttributeException("Invalid group value")
+        elif not subgroup:
+            raise InvalidSpreadsheetAttributeException("Invalid subgroup value")
+        else:
+            self._handler.add_row(self._student_sheet_title, [username, name, group, subgroup])
+
+    def remove_student(self, username: str) -> None:
```

Comment:
It is needed that this method return boolean value

Reply:
fixed